sirepo/package_data/raydata-execute-analysis.sh.jinja

```diff
@@ -0,0 +1,16 @@
+#!/bin/bash
+# No set -eou pipefail before activating the conda env because the
+# activation scripts have unbound variables and other errors that can
+# be ignored.
+if [[ -z "{{ conda_env }}" ]]; then
```

Comment:
This doesn't work right. For csx it becomes `if [[ -z "None" ]]; then` which evaluates to true.

Reply:
7407ed0e7849051a42a067015819229cd53c406b